MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+#### These are the current Htsjdk-Next-Beta project maintainers (sorted alphabetically by handle)
+
+-----
+
+* Jay Carey	**@jacarey** 	_jcarey@broadinstitute.org_
+*	Louis Bergelson	 **@lbergelson**	_louisb@broadinstitute.org_
+* Daniel Gómez Sánchez **@magicDGS** 	_daniel.gomez.sanchez@hotmail.es_
```

Comment:
Using `<>` around the email is more explicit for indicating that it is an email without rendering...